src/main.rs

```diff
@@ -56,6 +57,15 @@ fn encrypt(opt: cli::Opt, key: String, value: String) -> Result<()> {
     config.save(&opt.amber_yaml)
 }
 
+fn generate(opt: cli::Opt, key: String) -> Result<()> {
+    let value = sodiumoxide::randombytes::randombytes(16);
```

Comment:
I wonder if we should include a command line flag to control how many bytes of entropy are included here. What do you think?

Reply:
I was leaning towards the hardcoded value for simplicity and footgun-avoidance, but I guess there is a potential use-case when secrets of a speicific size are required? (Though the cases I can think of at the moment would also have their own preferred tools for generating & formatting the required values.)